src/main/java/com/lpnu/shaggybeavers/controller/CategoryController.java

```diff
@@ -0,0 +1,23 @@
+package com.lpnu.shaggybeavers.controller;
+
+import com.lpnu.shaggybeavers.service.CategoryService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/api/categories")
+@RequiredArgsConstructor
+public class CategoryController {
+
+    private CategoryService categoryService;
```

Comment:
every inejction recommended be final or use @Autowired , but recommended consturctor injection( make fileds final)